File: Cadastro de Pessoa/Cadastro de Pessoa/DTO/SexoDTO.cs
namespace ClienteREST.DTO
{
    public class SexoDTO : ObjetoDTO
    {
        public string descricao { get; set; }

        public SexoDTO() { }

        public SexoDTO(int id, string descricao)
        {
            this.id = id;
            this.descricao = descricao;
        }
    }
}

File: servico_basico/src/java/dto/SexoDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package dto;

import modelo.*;

/**
 *
 * @author superalunocmc
 */
public class SexoDTO extends ObjetoDTO {
    
    private String descricao;

    public SexoDTO() {
    }

    public SexoDTO(String descricao, int id) {
        super(id);
        this.descricao = descricao;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
    
    public static SexoDTO getDto(Sexo sexo){
        return new SexoDTO(
                sexo.getDescricao(),
                sexo.getId()
        );
    }
    
    public Sexo getObj(){
        return new Sexo(
                this.getDescricao(),
                this.getId()
        );
    }
}
